### DISCUSSION TITLE
Scrollable Panels

### DISCUSSION THREAD

**Discussion by parvathy-mpac** [bvaughn/react-resizable-panels#429] [Q&A] [answered]
Lets say we have a PanelGroup with direction 'vertical' with 2 panels inside of it -  Panel A and Panel B. We use the PanelResizeHandle and now Panel B covers most of Panel A. Does the library provide anything to make the Panels scrollable in such cases?

**Comment by bvaughn** [comment, marked as answer]
You can use CSS `overflow` prop (perhaps along with (`min-height` style for your panel's inner-content) to make it scrollable if it's resized below a certain size. That's nothing this Library does or helps with though. All CSS.
